includes/html/pages/bill.inc.php

```diff
@@ -9,7 +10,9 @@
     include 'includes/html/pages/bill/actions.inc.php';
 }
 
-if (bill_permitted($bill_id)) {
+if (!Bill::find($bill_id)) {
+    echo "Bill does not exist";
```

Comment:
Might be better to call `abort(404);`

Reply:
I've updated the query now :)

on the abort 404, happy to do that the only thing I'd say is that then just does a generic 404 page so you have to navigate back to get back to lnms. I've updated to use it but it feels better to keep people on a familiar page imho.